terraform/oke/schema.yaml

```diff
@@ -191,8 +198,8 @@ variables:
   # Option to create Dynamic Group and Policies
   opt_create_dynamicGroup_and_policies:
     type: boolean
-    title: Check to create Dynamic Group and Policies required for deploying monitoring solution
+    title: Select this check box to create dynamic groups and policies that are required for deploying the monitoring solution.
```

Comment:
link to details of which dynamic groups and policies along w/ corresponding policies